packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -154,11 +165,10 @@ function PostLockedModal() {
 		return null;
 	}
 
-	// Avoid sending the modal if sync is supported, but retain functionality around locks etc.
-	if ( window.__experimentalEnableSync && supportsSync ) {
-		if ( globalThis.IS_GUTENBERG_PLUGIN ) {
-			return null;
-		}
+	// If both the current user and the lock owner can collaboratively edit, do not
```

Comment:
Unless I'm misinterpreting the code, isn't this checking the user against the permissions of the post type since `syncEnabled` is checking for the experimental flag and the config of the post type?

Reply:
Yes. If the post type does not support collaborative editing, you also cannot collaboratively edit.